functions/modules/user/validation.js

```diff
@@ -1,3 +1,10 @@
-exports.validateUser = function (uid) {
-    return typeof uid !== 'undefined' && uid
+const pino = require("../../logger-setup");
```

Comment:
Simple suggestion mais qu'est-ce que tu penses de le nommer en `logger` ou `Logger` là où on l'import?
```suggestion
const logger = require("../../logger-setup");
```

Reply:
Ca me va, vendu